src/pages/ThorChainLP/queries/hooks/useAllUserLpData.ts

```diff
@@ -40,34 +34,25 @@ export const useAllUserLpData = ({
   const runeMarketData = useAppSelector(state => selectMarketDataById(state, thorchainAssetId))
 
   const { data: allThornodePools, isSuccess: isThornodePoolsDataLoaded } = useQuery({
```

Comment:
update variable to `availableThornodePools` for clarity

Reply:
2a7b7acbec73104067d60468da5ea1d381e6c6b4